homeassistant/components/recorder/util.py

```diff
@@ -402,10 +402,10 @@ def _datetime_or_none(value: str) -> datetime | None:
 def build_mysqldb_conv() -> dict:
     """Build a MySQLDB conv dict that uses cisco8601 to parse datetimes."""
     # Late imports since we only call this if they are using mysqldb
-    # pylint: disable=import-outside-toplevel,import-error
+    # pylint: disable-next=import-outside-toplevel,import-error
     from MySQLdb.constants import FIELD_TYPE
 
-    # pylint: disable=import-outside-toplevel,import-error
+    # pylint: disable-next=import-outside-toplevel,import-error
```

Comment:
I would combine these into one `pylint: disable` for the whole block.

Reply:
Done